lib/acpi/Kconfig

```
@@ -34,6 +34,11 @@ config ACPI_SHELL
 	help
 	  Enable commands for debugging ACPI using the built-in shell.
 
+config ACPI_POWEROFF
+	bool "Power off functionality by setting system to S5 power state"
+	help
+	Enable support for system power off through ACPI.
```

Help text needs a tab and two spaces indentation.

```suggestion
	  Enable support for system power off through ACPI.
```